src/manual/using/administration.md

```diff
@@ -41,10 +41,19 @@ administrative UI.
 
 Projects of various size and level of support make use Taskcluster and are managed in very different ways. The general rules below explain the responsibilities of various teams in managing access controls.
 
-## Small projects (experiments, young products, etc.)
+## Unmanaged projects (experiments, young products, etc.)
 
-Small projects are self-serve and the developers are responsible for managing their own configuration, secrets, etc. They are often hosted on GitHub and need to be initially created by a member of the Taskcluster team before being handed off to the developers. The following rules apply:
+Unmanaged projects are self-serve and the developers are responsible for managing their own configuration, secrets, etc. They are often hosted on GitHub and need to be initially created by a member of the Taskcluster team before being handed off to the developers. The following rules apply:
 
 * Developers are autonomous and responsible for managing access controls, scopes and permissions securely.
 * Taskcluster team provides administrative support to bootstrap the project
 * Release engineering is not involved until the project "graduate" and is moved to their purview
+
+## Managed projects (core products, etc.)
+
+* Taskcluster team manages the taskcluster service. It has permissions to create new projects and delegates permissions to
+    - Developers for unmanaged projects
+    - Release engineering team for managed projects
+* Taskcluster team drops its privileges to managed projects once they are handed over to release engineering. At this point, making a change to a managed project requires releng sign off.
```

Comment:
The taskcluster team has star (superuser / god) privs over the current cluster. We can make a social rule or policy that they won't touch these projects, but they will have the technical ability to make changes here still.

Reply:
We could theoretically get rid of that sooner rather than later. However, we would still have access to the root creds themselves I guess.

Or if we really wanted to commit, we could give admin of the heroku services over to another team as well and then we would be entirely outside the loop.

Reply:
I think it would make more sense to make that change when Firefox moves to its own cluster, but if we needed that change sooner, we could go that route or possibly via managed-scopes-monitoring.

Reply:
> We can make a social rule or policy

I think this is the goal. Until we have actual limitations, we can audit changes programmatically, and if someone crosses a line, raise an alert and investigate.

> we could give admin of the heroku services over to another team

This may be the future we're heading for with cloudops, right?

Reply:
>  I think this is the goal. Until we have actual limitations, we can audit changes programmatically, and if someone crosses a line, raise an alert and investigate.

Sounds good. Maybe we should change

```
Taskcluster team drops its privileges to managed projects once they are handed over to release engineering. At this point, making a change to a managed project requires releng sign off.
```

to something like

```
Once handed over to Release Engineering, making a change to a managed project requires releng sign off.
```

or

```
Taskcluster team shouldn't make unapproved changes to managed projects once they are handed over to Release Engineering...
```

Reply:
> This may be the future we're heading for with cloudops, right?

I think so, but we're referring to that as the 2nd cluster.